test/suite/test_bug019.py

```diff
@@ -75,9 +75,11 @@ def prepfiles(self):
     # There was a bug where pre-allocated log files accumulated on
     # Windows systems due to an issue with the directory list code.
     def test_bug019(self):
+        start_prealloc = self.get_prealloc_stat()
         # Create a table just to write something into the log.
+        # This should increase the max number of log files preallocated,
+        # as indicated by the max_prealloc statistic.
```

Comment:
Minor nit: the `max_prealloc` is a local variable in the test. The statistic itself is called `log_prealloc_max`. I suggest just removing that and having `as indicated by the statistic.`

Reply:
Thanks, done.